Nos/Service/CurrentUser.swift

```diff
@@ -194,7 +194,7 @@ import Dependencies
             kinds: [.contactList],
             limit: 2, // small hack to make sure this filter doesn't get closed for being stale
             since: author.lastUpdatedContactList,
-            keepSubscriptionOpen: true
+            keepSubscriptionOpen: false
```

Comment:
Since `keepSubscriptionOpen` defaults to `false`, you can simply remove this line.